spec.html

```diff
@@ -45021,6 +45023,7 @@ <h1>
               1. Assert: _result_.[[Type]] is ~throw~.
               1. Return ? _result_.
             1. Return CreateIterResultObject(_resultValue_, *true*).
+          1. Set the code evaluation state of _genContext_ such that when evaluation is resumed for that execution context, _closure_ will be invoked.
```

Comment:
I think we could maybe use a better term than "invoked" here which clarifies that there are no parameters expected/passed. We usually use "perform" for that when used with AOs.

Reply:
> I think we could maybe use a better term than "invoked" here

I think I used the word "invoked" because that's what the definition of Abstract Closures uses:
"Abstract Closures are meta-values and are invoked using function application style ..."

> which clarifies that there are no parameters expected/passed

We could say "invoked with no arguments".

> We usually use "perform" for that when used with AOs.

If we wanted a step to carry out a particular invocation (and we didn't care about the result), we would certainly say "Perform", but in a context like this, I think the spec is more inclined to use "call", e.g.:
- "Each time this operation [ResolveExport] is called ..."
- "Any subsequent call to HostResolveImportedModule ..."
- "calling the abstract operation thisNumberValue/thisBigIntValue/thisTimeValue ..."
- "this function calls ToNumber"
- "calling NotifyWaiter with arguments `_WL_` and `_W_`"

which suggests that we say "... `_closure_` will be called with no arguments."

Reply:
"called with no arguments" works for me

Reply:
> "called with no arguments" works for me

Done!